test/suites-ostree/isolated-network.robot

```diff
@@ -0,0 +1,46 @@
+*** Settings ***
+Documentation       Tests related to MicroShift running in an isolated network
+
+Resource            ../resources/ostree-health.resource
+Resource            ../resources/microshift-process.resource
+Resource            ../resources/microshift-host.resource
+Library             Collections
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+Test Tags           network
+
+
+*** Test Cases ***
+Verify Healthy Then Clean And Reboot MicroShift
+    [Documentation]    Verify no Internet access, check MicroShift health, clean
+    ...    MicroShift data and verify health again after rebooting the host
+
+    Verify No Internet Access
+    Wait For Healthy System
+
+    Cleanup MicroShift
+    Enable MicroShift
+    Reboot MicroShift Host
+
+    Verify No Internet Access
+    Wait For Healthy System
```

Comment:
It would be good to run some functional tests for things like services to verify that OVN-K is directing incoming traffic. Those tests might have to access the service from the host, I'm not sure.